src/common/CommonConfirmationPage/CommonConfirmationPage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FYALogo from '../../assets/fya-text-logo.svg';
+import FYALogoMobile from '../../assets/fya-logo-large.svg';
+import styles from './CommonConfirmationPage.module.css';
+
+const CommonConfirmationPage = ({ isOpen, children, showFYALogo }) => {
+  if (isOpen) {
+    return (
+      <div className={styles['confirmation-page']}>
+        {showFYALogo && <img src={FYALogo} className={styles.logo} alt="logo" />}
+        {showFYALogo && <img src={FYALogoMobile} className={styles['logo-mobile']} alt="logo" />}
+        {children}
```

Comment:
Can we add more common components to this CommonConfirmationPage? For example, the CheckCircleIcon is used in all the confirmation pages. We can also move the title and message Text components here and then just add props for the actual content for these (e.g. we would just pass in "Email Sent" and "A link to reset your password has been sent to your registered FYA email address" for the ForgotPasswordConfirmation).

Reply:
The Return to Login button can also be moved here if we are only using this CommonConfirmationPage for auth pages.

Reply:
I went ahead and pretty much added everything including the "Return to Login" button to the CommonConfirmationPage for now. I figured that if we needed to use this page elsewhere later we can just get ride of the button. Also I replaced the blue auth page backgrounds with their appropriate images but the styling is off, so I'll continue to work on that.